packages/bpk-component-pagination/src/BpkPaginationPage.module.scss

```diff
@@ -18,11 +18,13 @@
 
 @import '~bpk-mixins/index';
 
+$bpk-spacing-v2: true;
+
 .bpk-pagination-page {
-  min-width: $bpk-spacing-xl;
-  height: $bpk-spacing-xl;
+  min-width: bpk-spacing-xl() + bpk-spacing-sm();
```

Comment:
Is it worth including a comment to explain why we use this combo? (ie to match the height of the button)

Reply:
I think it's worth having this info in Figma perhaps as it's a design decision rather than having it in our code?

Reply:
Plus one to @anambl comment - given this is something from design rather than us and to keep in line with our own linting tools (instead of using a raw value), this makes sense.

As a lot of the changes might be just 'because that is the design given' vs. having a specific reason as to why we do this in code. We are just 'codifying the design language' and not necessarily having a reason for making a combo